gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -666,6 +666,12 @@ gst_tensor_filter_transform (GstBaseTransform * trans,
   out_flexible =
       gst_tensor_pad_caps_is_flexible (GST_BASE_TRANSFORM_SRC_PAD (trans));
 
+  if (priv->prop.invoke_dynamic && !out_flexible) {
```

Comment:
In the case of (priv->prop.invoke_dynamic && !in_flexible), is confirmation required?

Reply:
I will check your comment in next PR. Thanks.